lib/plugins/features/movies/controller.js

```diff
@@ -9,6 +11,20 @@ exports.create = (payload) => {
   });
 };
 
+exports.addLocation = (movie_id, payload) => {
+  return Bluebird.all([
+    new Movie({ id: movie_id }).fetch(),
+    new Location({ id: payload.id }).fetch()
+  ])
+  .spread((movie, location) => {
+    return movie.related('locations').attach(location);
+  })
+  .then(() => {
+    return new Movie({ id: movie_id })
+	       .fetch({ withRelated: Movie.RELATED });
```

Comment:
nit: in this case, you could probably put this all on one line:

```js
return new Movie({ id: movie_id }).fetch({ withRelated: Movie.RELATED });
```

but as a rule of thumb, if the continuing function is a promise function (e.g. `.then`, `.catch`, `.finally`, `.spread`, `.map`, etc), then we align the `.functionName` with the previous line:

```js
return new Movie({ id: movie_id }).fetch({ withRelated: Movie.RELATED })
.then(() => {
  // ...
})
```

and we also have some precedent of indenting non-promise functions one level in:

```js
return new Movie({ id: movie_id })
  .fetch({ withRelated: Movie.RELATED });
```